azure/template.json

```diff
@@ -687,6 +687,10 @@
                                 "name": "WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION",
                                 "value": "0"
                             },
+                            {
+                                "name": "WEBSITE_HEALTHCHECK_MAXPINGFAILURES",
+                                "value": "5" //At 1 ping per minute
```

Comment:
I don't think you can add a comment in json

Reply:
Apparently in ARM templates you can: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-syntax#comments